openeo/rest/job.py

```diff
@@ -25,10 +26,19 @@ class JobLogEntry:
     A Job log entry.
     """
 
-    def __init__(self, log_id: str, level: str, message: str):
-        self.log_id = log_id
-        self.level = level
-        self.message = message
+    def __init__(self, entry):
+        self.log_id = entry['id']
+        self.code = entry['code'] if 'code' in entry else 0
```

Comment:
in Python you can write this more succinctly:

    self.code = entry.get("code", 0)

second argument is fallback value if the key is not in the dict

Reply:
Sounds good, fixed.